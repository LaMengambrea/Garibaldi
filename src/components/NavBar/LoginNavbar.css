.LoginNavbar {
    background-color: #000000;
    color: #F2F2F2;
    padding: 5px;
    font-size: 0.9em;
}

.loginOption {
    float: right;
    margin-left: 20px;
    cursor: pointer;
}

.loginOption:hover{
    font-weight: bold;
}