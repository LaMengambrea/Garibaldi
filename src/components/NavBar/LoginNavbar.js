import React, {Component} from 'react';
import './LoginNavbar.css';

export default class LoginNavbar extends Component {
    render() {
        return (
            <div className="LoginNavbar">
                <div className="row">
                    <div className="col-xs-12 col-md-4">
                        <div className="loginOption">Regístrate</div>
                        <div className="loginOption">Iniciar sesión</div>
                    </div>
                    <div className="col-xs-12 col-md-4">
                        Aqui va el logo
                    </div>
                    <div className="col-xs-12 col-md-4"></div>
                </div>
            </div> 
        );
    }
}