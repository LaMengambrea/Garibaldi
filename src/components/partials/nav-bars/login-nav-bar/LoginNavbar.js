import React, {Component, PropTypes} from 'react';
import {Link} from 'react-router';
import './LoginNavbar.css';

export default class LoginNavbar extends Component {
    componentDidMount(){
        if(this.props.selectedOption){
            document.getElementById(this.props.selectedOption).style.fontWeight = "bold";
        }
    }

    render() {
        return (
            <div className="LoginNavbar">
                <div className="row">
                    <div className="col-xs-12 col-md-4">
                        <Link to="/" className="loginOption" id="loginOption">Iniciar sesión</Link>
                        <Link to="/register" className="loginOption" id="registerOption">Regístrate</Link>
                    </div>
                    <div className="col-xs-12 col-md-4">
                        <center>
                            <img src="" alt="" id="whiteNavLogo"/>
                        </center>
                    </div>
                    <div className="col-xs-12 col-md-4"></div>
                </div>
            </div> 
        );
    }
}

LoginNavbar.propTypes = {
    selectedOption: PropTypes.string
};