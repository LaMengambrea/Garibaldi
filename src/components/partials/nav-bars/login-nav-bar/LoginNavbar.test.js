import LoginNavbar from './LoginNavbar';
import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

let store = {
    subscribe: () => {},
    dispatch: () => {},
    getState: () => {
      return {
          selectedOption: ''
      };
    }
}

it('LoginNavbar renders', () => {
  const tree = renderer.create(
    <Provider store={store}>
            <LoginNavbar/>
    </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});