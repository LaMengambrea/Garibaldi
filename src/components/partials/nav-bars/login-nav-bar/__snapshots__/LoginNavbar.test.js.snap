exports[`test LoginNavbar renders 1`] = `
<div
  className="LoginNavbar row">
  <div
    className="col-xs-12 col-md-4">
    <a
      className="loginOption"
      id="loginOption"
      onClick={[Function]}
      style={Object {}}>
      Iniciar sesión
    </a>
    <a
      className="loginOption"
      id="registerOption"
      onClick={[Function]}
      style={Object {}}>
      Regístrate
    </a>
  </div>
  <div
    className="col-xs-12 col-md-4">
    <center>
      <img
        alt=""
        id="whiteNavLogo"
        src="" />
    </center>
  </div>
  <div
    className="col-xs-12 col-md-4" />
</div>
`;
